src/Symfony/Component/DependencyInjection/Loader/DirectoryLoader.php

```diff
@@ -40,6 +42,14 @@ public function load($file, $type = null)
         }
     }
 
+    /** {@inheritdoc} */
```

Comment:
please use multiple lines for this, to keep consistency with the code base (see the very next method e.g.)